src/test/feature/Ranger/rangerpolicy.py

```diff
@@ -96,12 +100,22 @@ def delete_policy(delete_policy_name, rangerhelper):
     delete_policy_name = options.deletedpolicyname
     
     #init rangerresthelper
-    helper = RangerRestHelper(host, port, rangeruser, rangerpasswd);
+    helper = RangerRestHelper(host, port, rangeruser, rangerpasswd)
     
     if new_policy_json_file_name != "":
-        policyname = create_policy(new_policy_json_file_name, helper)
-        print "policy {} created".format(policyname)
+        policyname, is_success = create_policy(new_policy_json_file_name, helper)
+        if is_success:
+            print "policy {} created".format(policyname)
+        else:
+            print "policy {} create failed".format(policyname)
+            sys.exit(-1)
```

Comment:
If add policy failed in ranger, the py script will return -1 instead of always return 0.